.github/workflows/release.yml

```diff
@@ -0,0 +1,186 @@
+name: Create release
+on: 
+  workflow_dispatch:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: Release to Github
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set_variables.outputs.version }}
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: "3.8"
+
+    - name: Install Hatch
+      run: pipx install hatch
+
+    - name: Set variables
+      id: set_variables
+      run: |
+        cp README.md plugin/
+        cd plugin/
+        metadata=$(hatch project metadata)
+        echo "::group::Variables"
+        cat << EOF | tee -a "$GITHUB_OUTPUT"
+        tag=$(git for-each-ref refs/tags --sort=-creatordate --format='%(refname:strip=2)' --count=1)
+        version=$(echo ${metadata} | jq -r .version)
+        project_name=$(echo ${metadata} | jq -r .name)
+        EOF
+        echo "::endgroup::"
+    - name: Bundle
+      if: github.event_name != 'pull_request' && format('refs/heads/{0}', github.event.repository.default_branch) == github.ref
```

Comment:
Can we move this conditional to the job level so we can skip the entire job instead?

Reply:
No. The docker job currently `needs` the job.